superset/viz.py

```diff
@@ -2367,7 +2382,7 @@ def query_obj(self):
             d["columns"] = gb
         return d
 
-    def get_js_columns(self, d):
+    def get_js_columns(self, d: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
`d` looks like a bad variable name. I thought these were forbidden in `.pylintrc`?

Reply:
@villebro I'm not sure if you're aware but numerous files have `pylint` checks and warnings (not errors) disabled including `viz.py` per the line `# pylint: disable=C,R,W`. Note the plan is to address this in [SIP-46](https://github.com/apache/incubator-superset/issues/9953). Note I'm not quite mentally prepared for that saga yet as the typing work has left me a tad battered and bruised.

Reply:
😄  I'm sure it has, kudos for this work! I think we need to bump `pylint` first, as the current version isn't compatible with `3.7`. I'm looking to see if there's interest in the community to help with that effort, as it might be a good task to get acquainted with the codebase.

Reply:
Agreed regarding bumping `pylint` (this is actually mentioned in the SIP) as I ran into an issue trying to use a newer feature. I'm not sure if we can automate this, but periodically I think it's good that we go through and bump all our dependencies which should help us be more nimble to accommodate us supporting more recent Python versions.